### PR TITLE
update the integration and NSE docs

### PR DESCRIPTION
**Pull request by zingale**
These were out of sync in a lot of places.  Also, now all of the runtime parameters are documented.

### CODE REVIEW

File: sphinx_docs/source/integrators.rst [hunk 1]
```diff
@@ -105,30 +105,50 @@ passed into the integration routines.  For this reason, we often need
...
 
-#. convert back to a ``burn_t`` by calling ``integrator_to_burn``
+#. Recompute the temperature if we are using ``integrator.call_eos_in_rhs``.
 
-#. normalize the abundances so they sum to 1.
+#. If we set ``integrator.subtract_internal_energy``, then subtract
+   off the energy offset, the energy stored is now just that generated
+   by reactions.
+
+#. Normalize the abundances so they sum to 1.
```
Comment (zhichen3) [inline sphinx_docs/source/integrators.rst:141]: mention we only normalize when we don't set ``integrator.use_number_density``?

File: sphinx_docs/source/nse.rst [hunk 2]
```diff
@@ -86,35 +96,39 @@ NSE Table Outputs
 -----------------
 
 The NSE table provides values for the auxiliary composition,
-:math:`Y_e`, :math:`\bar{A}`, and :math:`\langle B/A \rangle`
+:math:`\bar{A}`, and :math:`\langle B/A \rangle`
 resulting from the full 125 nuclei network.   It also provides a set of 19
```
Comment (zhichen3) [inline sphinx_docs/source/nse.rst:100]: 125 -> 96

File: sphinx_docs/source/nse.rst [hunk 3]
```diff
@@ -129,34 +143,56 @@ The basic flow of a simulation using ``aprox19`` + the NSE table is as follows:
...
-    * :math:`\bar{A}_{\rm out}` is simply the value returned from the table
-
-    * the energy generation rate, :math:`e_{\rm nuc}` is:
+      In particular, the energy source is constructed as:
 
       .. math::
 
-         e_{\rm nuc} = \eta \left [ \left ( \frac{B}{A} \right )_{\rm out} -
-                                    \left ( \frac{B}{A} \right )_{\rm in} \right ] * \frac{1.602 \times 10^{-6}  {\rm erg}}{{\rm MeV}} N_A \frac{1}{\Delta t}
+         R(\rho e) = N_A \frac{\Delta (\rho \langle B/A\rangle)}{\tau} + N_A \Delta m_{np} c^2 \rho \frac{dY_e}{dt} - \rho (\epsilon_{\nu,\mathrm{thermal}} + \epsilon{\nu,\mathrm{react}})
```
Comment (zhichen3) [inline sphinx_docs/source/nse.rst:166]: missing underscore after \epsilon in `\epsilon{\nu,\mathrm{react}}`

File: sphinx_docs/source/nse.rst [hunk 4]
```diff
@@ -129,34 +143,56 @@ The basic flow of a simulation using ``aprox19`` + the NSE table is as follows:
...
+
+      where $\Delta m_{np}$ is the difference between the neutron and H atom mass.
+
+      .. important::
+
+         It only makes sense to include the weak rate neutrino losses, $\epsilon{\nu,\mathrm{react}}$,
```
Comment (zhichen3) [inline sphinx_docs/source/nse.rst:172]: missing underscore here too

File: sphinx_docs/source/nse.rst [hunk 5]
```diff
@@ -129,34 +143,56 @@ The basic flow of a simulation using ``aprox19`` + the NSE table is as follows:
...
+         if the initial model that you are using in your simulation also included those losses.
+         Otherwise, the energy loss from our NSE table will likely be too great and that simulation
+         will not be in equilibrium.  This is an issue, for example, when using a MESA model
+         constructed with ``aprox21``, which does not have all of the weak rates we model here.
 
+         The weak rate neutrino losses can be disabled by ``integrator.nse_include_enu_weak``.
```
Comment (zhichen3) [inline sphinx_docs/source/nse.rst:178]: perhaps specify to disable, set `integrator.nse_include_enu_weak=0`

File: sphinx_docs/source/nse.rst [hunk 6]
```diff
@@ -129,34 +143,56 @@ The basic flow of a simulation using ``aprox19`` + the NSE table is as follows:
...
+         \Uc^{\prime,n+1/2} = \Uc^{\prime,n} + \frac{\Delta t}{2} [\Advs{\Uc^\prime}]^{n+1/2} + \frac{\Delta t}{2} [\Rb(\Uc^\prime)]^{n+1/2}
 
-    * update the mass fractions, :math:`X_k`, using the values from the table
+
+      where $[\Advs{\Uc^\prime}]^{n+1/2}$ are the advective updates carried by the SDC
+      algorithm.
+
+    * Compute the energy generation rate from the change in internal energy from $\Uc^{\prime,n}$ to $\Uc^{\prime,n+1}$, excluding advection.
+
+    * Update the total energy.
```
Comment (zhichen3) [inline sphinx_docs/source/nse.rst:196]: do we mention we get the new massfractions by calling the nse-table with temperature computed from the updated energy and updated Ye?

File: sphinx_docs/source/integrators.rst (hunk 1 above)
Comment (zingale) [reply sphinx_docs/source/integrators.rst]: done

File: sphinx_docs/source/nse.rst (hunk 2 above)
Comment (zingale) [reply sphinx_docs/source/nse.rst]: fixed

File: sphinx_docs/source/nse.rst (hunk 3 above)
Comment (zingale) [reply sphinx_docs/source/nse.rst]: fixed

File: sphinx_docs/source/nse.rst (hunk 6 above)
Comment (zingale) [reply sphinx_docs/source/nse.rst:196]: added

File: sphinx_docs/source/nse.rst (hunk 4 above)
Comment (zingale) [reply sphinx_docs/source/nse.rst]: fixed

**Comment by zingale** [comment]
I think I addressed all the comments.

**Review by zhichen3 — APPROVED** [review]

**Pull request merged**